sources/Common/Dictionaries/DYNAlgorithmsError_en_GB.dic

```diff
@@ -18,3 +18,4 @@ DirectoryDoesNotExist            = directory : %1% does not exist
 XmlParsingError                  = error while parsing file %1% : %2%
 IncoherentAccuracy               = accuracy of margin calculation should be a number between 1 and 100 (found : %1%)
 InputFileFormatNotSupported      = input file should be either a zip or a xml file (found %1%)
+LoadIncreaseResultNotFound       = load increase result of variation %1% is not found
```

Comment:
SHould be in french dictionnary too

Reply:
- [ ] to be done if this exception is kept

Reply:
Eventually this exception is not kept in the code.

Reply:
This error doesn't exist anymore.